spring-integration-core/src/main/java/org/springframework/integration/dispatcher/PartitionedDispatcher.java

```diff
@@ -165,11 +164,12 @@ public void shutdown() {
 	}
 
 	@Override
+	@SuppressWarnings("NullAway")
```

Comment:
Can we state in the comment to this annotation the reason?
Something similar to what I suggested in the review, but might be more specific to this situation.